scrapers/tv.py

```
@@ -166,3 +171,60 @@ async def validate_and_update_tv_stream(stream, bulk_writer):
     validate_tv_streams_in_db.send_with_options(
         args=(page + 1, page_size), delay=2 * 6000
     )
+
+
+@dramatiq.actor(time_limit=30 * 60 * 1000, priority=5)
+async def update_tv_posters_in_db(*args, **kwargs):
+    """Validate TV posters in the database."""
+    not_working_posters = await MediaFusionTVMetaData.find(
+        MediaFusionTVMetaData.is_poster_working == False
```

Ruff

Avoid equality comparisons to `False`; use `if not MediaFusionTVMetaData.is_poster_working:` for false checks

Replace with `not MediaFusionTVMetaData.is_poster_working`

(E712)